### DISCUSSION TITLE
Adding a timestamp to my videos

### DISCUSSION THREAD

**Discussion by kaylamac35** [bonsai-rx/bonsai#2048] [Q&A]
[MA Videos.zip](https://github.com/user-attachments/files/17593686/MA.Videos.zip)

<img width="249" alt="image" src="https://github.com/user-attachments/assets/03e4b946-454c-4594-8870-a8968cbcf3e9">

I am trying to create a video where I can record for long periods of time, but every 10 minutes of so, the video will be uploaded to the cloud so I can look in real time. This workflow works how I want it to, but now I want to add timestamps on the video itself. How could I do this?

**Comment by sannatitus** [comment]
Hiya! If you're using video writer node to save your videos, there's an option in the right-side menu to add a file suffix of a timestamp

**Comment by banchan86** [comment]
Do you mean you want to overlay timestamps on the video itself? Not sure if this is the best solution but you could do something like this by using the Bonsai.Vision package.


<img src="https://github.com/user-attachments/assets/8fc132fd-22e7-488b-ac67-cc06f07fda84" width="500">

Please see attached workflow as well.

[#2048_amended.zip](https://github.com/user-attachments/files/18188974/2048_amended.zip)
